kong/init.lua

```diff
@@ -1643,6 +1644,15 @@ function Kong.admin_header_filter()
   --ctx.KONG_ADMIN_HEADER_FILTER_TIME = ctx.KONG_ADMIN_HEADER_FILTER_ENDED_AT - ctx.KONG_ADMIN_HEADER_FILTER_START
 end
 
+function Kong.admin_gui_kconfig_content()
+  local content = kong.cache:get(
```

Comment:
`kong.cache.get` may also throw an error. I think need to handle it to get better.

Reply:
Another question is when do we refresh the cache data?

Reply:
It seems that the parameters in `kconfig.js` are read and used at `kong start`, and tend to keep unchanged at runtime.

https://github.com/Kong/kong/blob/f31037a261887c1400b50d3b49ff72b4f1b0a923/kong/admin_gui/init.lua#L34-L42

It is ok, provided the current config parameters are stable at runtime. However, we'd better double-check. cc @nekolab